config/manufacturer_specific.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<ManufacturerSpecificData Revision="133" xmlns="https://github.com/OpenZWave/open-zwave">
+<ManufacturerSpecificData xmlns="https://github.com/OpenZWave/open-zwave">
```

Comment:
`Revision="134"` needs to be placed in here.

Reply:
Done

Thanks again for the support!